src/components/ActivityLog/FilterDateRange.tsx

```diff
@@ -66,8 +67,8 @@ export default function DateRange({ field, onChange, onDelete, values }: DateRan
   );
 
   return (
-    <Box alignItems='center' display='flex' flexDirection='row' flexWrap='wrap' sx={{ marginBottom: theme.spacing(3) }}>
-      <Box alignItems='center' display='flex' flexDirection='row'>
+    <Box alignItems='center' display='flex' flexDirection='row' flexWrap='wrap'>
+      <Box display='flex' flexDirection='row' justifyContent='center' marginBottom={theme.spacing(2)}>
```

Comment:
Can we add the margin only for mobile? I just added the pills and there is too much space between the datepickers and the pills with this